charts/community-operator/values.yaml

```diff
@@ -1,3 +1,9 @@
+## Reference to one or more secrets to be used when pulling images
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+##
```

Comment:
Could you please remove this empty comment? This way the comments throughout the file will look symmetric.

Reply:
removed